CWP/papers/roadmap/authors/author_stub.tex

```diff
@@ -0,0 +1,10 @@
+% Stub to test the latex processing of the author list
+
+\documentclass[12pt,a4paper]{article}
+
```

Comment:
As arXiv uses pdflatex we need 

\usepackage[utf8]{inputenc}

Reply:
You are right, I stopped using the stub when you switch to pdflatex... I'll fix it.